bootstrap/geyser/src/main/java/com/rtm516/mcxboxbroadcast/bootstrap/geyser/MCXboxBroadcastExtension.java

```diff
@@ -126,7 +126,37 @@ public void onPostInitialize(GeyserPostInitializeEvent event) {
 
             // Start the update timer
             GeyserImpl.getInstance().getScheduledThread().scheduleWithFixedDelay(this::tick, config.updateInterval, config.updateInterval, TimeUnit.SECONDS); // TODO Find API equivalent
+
+            // Due to API limitations stated in the config, need a separate update timer
+            GeyserImpl.getInstance().getScheduledThread().scheduleAtFixedRate(this::autoFriend, config.friendSyncConfig.updateInterval, config.friendSyncConfig.updateInterval, TimeUnit.SECONDS);
```

Comment:
This should only be added if one of the options is enabled. See https://github.com/rtm516/MCXboxBroadcast/blob/master/bootstrap/standalone/src/main/java/com/rtm516/mcxboxbroadcast/bootstrap/standalone/StandaloneMain.java#L101